internal/bloblang/query/functions.go

```diff
@@ -699,6 +700,24 @@ var _ = registerSimpleFunction(
 
 //------------------------------------------------------------------------------
 
+var _ = registerSimpleFunction(
+	NewFunctionSpec(
+		FunctionCategoryGeneral, "snowflake_id",
+		"Generate a new snowflake ID each time it is invoked and prints a string representation.",
+		NewExampleSpec("", `root.id = snowflake_id()`),
+	),
+	func(_ FunctionContext) (interface{}, error) {
+		node, err := snowflake.NewNode(1)
```

Comment:
I guess some users might want to pass in a custom node number, but not sure. Maybe consider having it as an optional parameter with `1` as the default value

Reply:
Thanks for the suggestion, it's done ✅